### ISSUE TITLE
NPE when calling Reflections#collect(String, Predicate<String>, Serializer)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#109]
```
What steps will reproduce the problem?
- running maven clean test
or:
- run ReflectionsTest

What is the expected output? What do you see instead?
- all Tests should all pass

What version of the product are you using? On what operating system?
- current trunk with commit #150

Please provide any additional information below.
A forked repository can be found at 
https://github.com/gesellix/reflections-forked where you will find a branch 
"develop" containing a fix for the NPE. Additionally, you will find a patch 
file attached to this issue.
```

Original issue reported on code.google.com by `gesellix` on 23 Apr 2012 at 2:15

Attachments:
- [ReflectionsCollectNPE.patch](https://storage.googleapis.com/google-code-attachments/reflections/issue-109/comment-0/ReflectionsCollectNPE.patch)

**Comment by GoogleCodeExporter** [comment]
```
fixed on 0.9.7-RC1 and trunk. comments?
```

Original comment by `ronm...@gmail.com` on 2 May 2012 at 3:43
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Great, I'll give it a try. Thanks!

```

Original comment by `gesellix` on 3 May 2012 at 9:51

**Comment by GoogleCodeExporter** [comment]
```
as followup please have a look at issue #111, a fix is provided as patch (using 
code from an older revision).
```

Original comment by `gesellix` on 4 May 2012 at 6:43

**Comment by GoogleCodeExporter** [comment]
```
fixed 0.9.7
```

Original comment by `ronm...@gmail.com` on 22 May 2012 at 7:47
- Changed state: **Fixed**
